prover/README.md

```diff
@@ -0,0 +1,30 @@
+## How-to run Snarkify prover locally
+1. `make snarkify`
+2. update `config.json` with right coordinator and l2geth endpoints
```

Comment:
nit: better to list what are the endpoints for testnet and mainnet.

Reply:
and why prover needs coordinator endpoint? My gut is that only relayer needs it.

Reply:
It doesn't use coordinator endpoint, I'll remove this requirement for now.